__tests__/helpers.ts

```diff
@@ -24,10 +24,6 @@ export {
   WEEK,
 } from "jest-time-helpers";
 
-// Sometimes CI's clock can get interrupted (it is shared infra!) so this
-// extends the default timeout just in case.
-jest.setTimeout(15000);
```

Comment:
This had to be removed because the `jest` global isn't available in the global setup environment, which utilizes this module. It doesn't seem like the ideal place for it anyway. Instead I changed the `testTimeout` option in the config file.